src/dvc_render/vega_templates.py

```diff
@@ -595,6 +583,38 @@ class SmoothLinearTemplate(Template):
                     "color": {"field": "rev", "type": "nominal"},
                 },
             },
+            {
+                "transform": [
+                    {
+                        "calculate": "datum.rev + '::' + datum.filename + '::' + datum.field",  # noqa: E501
```

Comment:
These fields have to be present in the data in order to implement the new tooltips. I am not sure if these or `dvc_data_version_info` are present in the Studio/DVC data. Will need to investigate.

Reply:
This line is also too long any giving pylint the shits.

Reply:
From https://github.com/iterative/dvc/pull/9931#issuecomment-1714195951 seems like we want to add an actual ID field instead of calculating this.